scenario_player/main.py

```diff
@@ -228,18 +236,27 @@ def run(
     if enable_ui:
         log_buffer = attach_urwid_logbuffer()
 
+    overrides = {
+        "spaas": {
+            "rpc": {
+                "port": service_port
+            }
+        }
+    }
+    scenario_definition = ScenarioDefinition(scenario_file, data_path, overrides)
+
     # Dynamically import valid Task classes from sceanrio_player.tasks package.
     collect_tasks(tasks)
 
     # Start our Services
-    service_process = ServiceProcess()
+    service_process = ServiceProcess(port=service_port)
```

Comment:
@nlsdfnbch Not sure i get your comment. You mean i should be using the `scenario_definition.spaas.rpc.port` here?

Reply:
Yupp, that's what I'm saying.

Reply:
Done.